src/server/services/decos/helpers.ts

```diff
@@ -60,7 +63,7 @@ export function isScheduledForRemoval(decosZaakSource: DecosZaakSource) {
 
 export function isExcludedFromTransformation(
   zaakSource: DecosZaakSource,
-  zaakTypeTransformer: DecosZaakTypeTransformer
+  zaakTypeTransformer: DecosZaakTransformer<any>
 ) {
   return (
```

Comment:
Je kunt in deze functie ook dit doen (zie helpers.ts): `isWaitingForPaymentConfirmation(zaakSource, zaakTypeTransformer) ||`

Reply:
Bedankt. Had deze gemist!